src/com/facebook/buck/apple/BUCK

```diff
@@ -34,10 +34,13 @@ java_immutables_library(
     'GroupedSource',
     'ProvisioningProfileMetadata',
   ],
-  tests = [
-    '//test/com/facebook/buck/apple:apple',
-    '//test/com/facebook/buck/apple:apple_test_integration',
-  ],
+#   We can't specify tests since we are using unsafe imports and
+#   targets might not be available on some platforms.
+#
+#   tests = [
+#     '//test/com/facebook/buck/apple:apple',
+#     '//test/com/facebook/buck/apple:apple_test_integration',
+#   ],
```

Comment:
Hmm, revert it from this PR, and open a new PR for this, we can figure out what is going separately.